script/Arcueid/Day 8/08_00_ARC08_8_1B.txt

```diff
@@ -140,6 +140,7 @@ Quality of her explanation aside, what she's talking about does ring a bell.
 [sha:2910cc7549526b9a3e0bf95f405450b33c03256d]{
 -- Page 553, Offset 21165.
 -- 志貴クンは無神論者でしょ？　だから実感は湧かないだろうけど、私たち教会は主の御名のもと、人の善き営みを守る意思がある。たとえそれが、こんな末法の国でもね」
+// group pr note: something about Noel voicing her displeasure to Japan, country about to fall soon or something
```

Comment:
Like I said in group PR: I feel pretty strongly that stripping this of its literal meaning is to take away a genuine allusion.

Petrikow seems to think that the allusion's meaning is all that matters and that it should just be "Even in a country as degenerate as this one" or something to that effect